src/herder/test/HerderTests.cpp

```diff
@@ -1368,51 +1365,47 @@ testSCPDriver(uint32 protocolVersion, uint32_t maxTxSize, size_t expectedOps)
 
         auto const lclCloseTime = lcl.header.scpValue.closeTime;
 
-        auto testTxBounds = [&](TimePoint const minTime,
-                                TimePoint const maxTime,
-                                TimePoint const nextCloseTime,
-                                bool const expectValid) {
-            REQUIRE(nextCloseTime > lcl.header.scpValue.closeTime);
-            // Build a transaction set containing one transaction (which could
-            // be any transaction that is valid in all ways aside from its time
-            // bounds) with the given minTime and maxTime.
-            auto tx = makeMultiPayment(root, root, 10, 1000, 0, 100);
-            auto& tb = tx->getEnvelope().type() == ENVELOPE_TYPE_TX_V0
-                           ? tx->getEnvelope().v0().tx.timeBounds.activate()
-                           : tx->getEnvelope().v1().tx.timeBounds.activate();
-            tb.minTime = minTime;
-            tb.maxTime = maxTime;
-            auto& sig = tx->getEnvelope().type() == ENVELOPE_TYPE_TX_V0
-                            ? tx->getEnvelope().v0().signatures
-                            : tx->getEnvelope().v1().signatures;
-            sig.clear();
-            tx->addSignature(root.getSecretKey());
-            auto txSet = std::make_shared<TxSetFrame>(
-                app->getLedgerManager().getLastClosedLedgerHeader().hash);
-            txSet->add(tx);
+        auto testTxBounds =
+            [&](TimePoint const minTime, TimePoint const maxTime,
+                TimePoint const nextCloseTime, bool const expectValid) {
+                REQUIRE(nextCloseTime > lcl.header.scpValue.closeTime);
+                // Build a transaction set containing one transaction (which
+                // could be any transaction that is valid in all ways aside from
+                // its time bounds) with the given minTime and maxTime.
+                auto tx = makeMultiPayment(root, root, 10, 1000, 0, 100);
+                setMinTime(tx, minTime);
+                setMaxTime(tx, maxTime);
```

Comment:
Changing these two lines for some reason causes clang-format to reformat this entire function.